core/controllers/base.py

```diff
@@ -339,6 +339,20 @@ def render_json(self, values):
         json_output = json.dumps(values, cls=utils.JSONEncoderForHTML)
         self.response.write('%s%s' % (feconf.XSSI_PREFIX, json_output))
 
+    def render_blob(self, blob):
```

Comment:
Why is this function still present (given our previous discussion)?

Reply:
Yes, not required. Removed.